etc/noserver.ini.in

```diff
@@ -23,6 +23,7 @@ pyramid.includes =
 tm.attempts = 5
 
 zodbconn.uri = memory://
+zodbconn.uri.audit = memory://auditlog
 
```

Comment:
nosever is only used by buildout to autogenerate typescript classed, 
no need for auditlog

But we should add all new config parameters in the example development init in  adhocracore_core.scaffold.--